packages/testsweets/lib/src/ui/driver_layout/driver_layout_view.dart

```diff
@@ -79,12 +77,12 @@ class _DriverLayoutViewState extends State<DriverLayoutView> {
           if (_showDebugInformation)
             ...model.descriptionsForView.map(
               (description) => Positioned(
-                top: description.position.y -
+                top: description.position!.y -
```

Comment:
No widget can exist without a position in our system. Our code should reflect that. This value should not be nullable.

Reply:
I don't remember why is that I'll see the code

Reply:
Yeah it was also about the view widget, we were assigning x=0 and y=0 to satisfy the constructor